tests/resources/TestCaseBody.robot

```diff
@@ -459,3 +459,28 @@ Check Harbor Api Page
     ${Title}=  Get Title
     Should Be Equal  ${Title}  Harbor Swagger
     Retry Wait Element  xpath=//h2[contains(.,"Harbor API")]
+
+Body Of Stop Scan And Stop Scan All
```

Comment:
Scan and Scan all should be encapsulated into Keywords.

Reply:
It has been modified